ApiHelper.php

```diff
@@ -66,10 +66,10 @@ private function connectDb() : void {
 		}
 
 		$this->db = new mysqli(
-			$this->creds['dbhost'],
-			$this->creds['dbuser'],
-			$this->creds['dbpass'],
-			'enwiki_p',
+                        $this->creds['dbhost'],
+                        $this->creds['dbuser'],
+                        $this->creds['dbpass'],
+                        $this->creds['db'],   /*enwiki_p,*/
```

Comment:
Indented too far and should use tabs

Reply:
Also no need for the `/* enwiki_p, */ comment

Reply:
done